code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -56,6 +56,13 @@
 		for(var/obj/structure/spider/S in range(1, get_turf(src)))
 			return S
 
+/mob/living/simple_animal/hostile/poison/giant_spider/death(gibbed)
```

Comment:
It might be simpler to just increment and decrement `SSmobs.giant_spiders` on `Initialize()`/`Destroy()` respectively.

Reply:
That does mean however, that you can just kill spiders until you have 40 dead bodies, and then they can't reproduce because `Destroy()` isn't called, which would be weird.

Reply:
Is there a better place do this if not the death proc?

Reply:
Right, giant spidders don't auto-gib like terrors do -- I'd probably keep `death()` then

Reply:
Alright sounds like a plan.